Dockerfile

```diff
@@ -16,6 +16,9 @@
 
 FROM codait/max-base:v1.1.3
 
+# Upgrade packages to meet security criteria
+RUN apt-get update && apt-get upgrade -y libgcrypt20 openssl perl wget apt curl e2fsprogs git patch && rm -rf /var/lib/apt/lists/*
```

Comment:
Why not upgrade everything? There is no harm to do so. Python shouldn't be managed by apt in this image.